src/main/kotlin/in/tazj/k8s/letsencrypt/acme/Route53Responder.kt

```diff
@@ -72,6 +72,17 @@ class Route53Responder(val route53: AmazonRoute53) : DnsResponder {
                 .reduce { acc, zone ->
                     if (zone.name.length > acc.name.length) {
                         zone
+                    } else if (zone.name.length == acc.name.length) {
```

Comment:
Should this compare the length or actually match the complete string? I haven't used private zones much, but I assume the actual name would be equal?